projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/commandsAndMenu.tsx

```diff
@@ -120,18 +122,36 @@ function add_command(
           setJobDataToDefault();
           runningVdkOperation = '';
         } else {
-          await showErrorMessage(
-            'Another VDK operation is currently running!',
-            'Please wait until the operation ends!',
-            [Dialog.okButton()]
-          );
+          await showDialog({
+            title: 'Error',
+            body: (
+              <div className="vdk-error-dialog">
```

Comment:
The same error dialog structure is repeated in multiple places. It's better if we create a reusable React component for displaying errors

Reply:
yeah, this makes a lot of sense to refactor. I will come up with proper showErrorDialog util function and use it where possible.

Reply:
Done.